RecoLocalTracker/Records/interface/SiStripClusterizerConditionsGPURcd.h

```diff
@@ -0,0 +1,13 @@
+#ifndef RecoLocalTracker_Records_SiStripClusterizerConditionsGPURcd_h
+#define RecoLocalTracker_Records_SiStripClusterizerConditionsGPURcd_h
+#include "FWCore/Framework/interface/DependentRecordImplementation.h"
+#include "FWCore/Utilities/interface/mplVector.h"
+
+#include "CalibTracker/Records/interface/SiStripDependentRecords.h"
+
+class SiStripClusterizerConditionsGPURcd : public edm::eventsetup::DependentRecordImplementation<
```

Comment:
for my own education, why a new record is needed?
cf:

https://github.com/cms-sw/cmssw/blob/5b56ea4f85b845ed3238c59310f79185ff481ab5/RecoLocalTracker/Records/interface/SiStripClusterizerConditionsRcd.h#L8-L11

Reply:
@mmusich I'm not an expert on the conditions system, but it was my understanding that the record declarations create the dependency graph that determines what needs to be updated when an IOV changes.  I didn't see an obvious way to piggy-back on the existing record dependencies

Reply:
OK, that's fine, because you have introduced a new CalibFormat: https://github.com/cms-sw/cmssw/pull/34618/files#diff-aac289cb93f5ee455be9217eafd63fb5f478fec2ffba1b75043d031b053369bb

Reply:
I'd like to understand the situation better.

> it was my understanding that the record declarations create the dependency graph that determines what needs to be updated when an IOV changes.

This is more or less correct. It follows that because `SiStripClusterizerConditionsRcd` and `SiStripClusterizerConditionsGPURcd` depend on the same records, they have identical IOV structure. Therefore, as far as the framework is concerned, replacing the use of `SiStripClusterizerConditionsGPURcd` with `SiStripClusterizerConditionsRcd` everywhere should work (since the data product type differs from anything already in `SiStripClusterizerConditionsRcd`). Am I missing something?

Reply:
I had a short look and agree with @makortel that the ```SiStripClusterizerConditionsGPURcd``` is redundant. I proposed a change to @dan131riley to eliminate it and reuse ```SiStripClusterizerConditionsRcd``` instead..